### ISSUE TITLE
Allow visibility of hostnames/IP addresses of nodes for non-admin users

### ISSUE THREAD

**Issue by smcvey** [cockroachdb/cockroach#77665]
**Is your feature request related to a problem? Please describe.**
v21.1.8 and in v20.2.14 removed the ability for non-admin users to see the hostname/IP addresses of nodes, this is still required for some customers.

**Describe the solution you'd like**
Update the `VIEWACTIVITY` role option so that when assigned to a role or user, they can see the full hostname/ip address of each node in the cluster:
* See hostnames/IPs appear in DB Console's Node overview page
* Access `_status/nodes` (currently gated by ADMIN)
* See hostnames/IPs in the `/reports/nodes` page (this page is currently viewable by non-ADMINs, but we obfuscate hostnames/IPs)

**Describe alternatives you've considered**
The alternatives are to make the user an admin, which is poor security practis]ce.

(edited to clarify solution) 

Jira issue: CRDB-13703

gz#11707

**Comment by knz** [comment]
This is in the DB console right?

**Comment by thtruo** [comment]
Based on internal conversations, we're considering updating the `VIEWACTIVITY` role option so that when it is assigned to a role or user, they can see the hostname and IP address of each node in the cluster within DB Console (under the node list in the overview page) 

@data-matt is that it? Currently, the `_status/nodes` endpoint and the `#/reports/nodes` page is gated to ADMINS. Is there a need to have these gated instead by the updated `VIEWACTIVITY` role option? 

cc @rafiss for awareness in case you have other solutions in mind

**Comment by data-matt** [comment]
@knz correct, we've hidden the hostnames in dbconsole behind the admin role.

**Comment by data-matt** [comment]
@thtruo will need to verify that.

**Comment by rafiss** [comment]
For context, this is the PR where the information was restricted to `admin`s: https://github.com/cockroachdb/cockroach/pull/67067

**Comment by dhartunian** [comment]
@thtruo can you clarify which versions this should be backported to?

**Comment by thtruo** [comment]
Yep, it should apply to both 21.1 and 21.2 - looks like the next scheduled patch release is 21.1.17 and 21.2.8
